cabal-install/src/Distribution/Client/ProjectConfig/Legacy.hs

```diff
@@ -356,7 +356,7 @@ commandLineFlagsToProjectConfig globalFlags NixStyleFlags {..} clientInstallFlag
                                   , packageConfigHaddockLinkedSource = packageConfigHaddockLinkedSource pc
                                   })
 
--- | Convert from the types currently used for the user-wide @~/.cabal/config@
+-- | Convert from the types currently used for the user-wide Cabal config
```

Comment:
Good way to avoid the explicit paths issue. Use it more widely?

Reply:
This is fine for an internal Haddock comment that will only be read by Cabal developers.  It is inappropriate for many of the other messages, which show up in CLI output, and may be read by novices.